firebase-database/ktx/src/main/kotlin/com/google/firebase/database/ktx/Database.kt

```diff
@@ -59,6 +67,66 @@ inline fun <reified T> MutableData.getValue(): T? {
     return getValue(object : GenericTypeIndicator<T>() {})
 }
 
+/**
+ * Starts listening to this query and emits its values via a [Flow].
+ *
+ * - When the returned flow starts being collected, a [ValueEventListener] will be attached.
+ * - When the flow completes, the listener will be removed.
+ */
+fun Query.snapshots() = callbackFlow<DataSnapshot> {
+    val listener = addValueEventListener(object : ValueEventListener {
+        override fun onDataChange(snapshot: DataSnapshot) {
+            repo.scheduleNow {
```

Comment:
hm, where is `repo` coming from? looks like it's `protected` in `Query`, I am surprised that it's accessible from here

Reply:
oh, actually nvm, it looks like there is a getter defined, but still lint complains about `@RestrictTo(LIBRARY_GROUP)`, can you pls update `database/ktx.gradle` like it's done for firestore https://github.com/firebase/firebase-android-sdk/blob/523730a3b4463160490170d9348818a5c403a2eb/firebase-firestore/ktx/ktx.gradle#L20 ?

Reply:
@vkryachko  Done